### ISSUE TITLE
after El Capitan brew update is verbose

### ISSUE THREAD

**Issue by smartinsightsfromdata** [Homebrew/homebrew#44549]
after El Capitan brew update is verbose. 

What to do to go back to a sort of silent install?

Now I see everything (small example):
````
M test/support/binaries/transmission-2.61.dmg
 M test/support/binaries/xarcontainer.xar
 M test/support/cleanup.rb
 M test/support/fake_dirs.rb
 M test/support/fake_fetcher.rb
 M test/support/fake_system_command.rb
 M test/support/never_sudo_system_command.rb
 M test/syntax_test.rb
 M test/test_helper.rb
Stashing your changes:
 M .travis.yml
 M CONTRIBUTING.md
 M README.md
 M a5.rb
 M abacas.rb
 M abinit.rb
 M abyss-explorer.rb
 M abyss.rb
 M ace-corrector.rb
 M adam.rb
 M adol-c.rb
 M alembic.rb
 M alglib.rb
 M allpaths-lg.rb
 M alpscore.rb
 M amos.rb
 M analysis.rb
 M andi.rb
 M ann.rb
````

**Comment by tdsmith** [comment]
Homebrew is reporting that these files seem to have been modified; you can check out a pristine version of the Homebrew formulas by running `cd $(brew --repository) && git reset --hard origin/master`. Let us know if this doesn't work or if we can help with anything else.

**Comment by smartinsightsfromdata** [comment]
Thanks for the prompt response, but sadly it didn't change a thing.

**Comment by tdsmith** [comment]
Please copy the entire output of brew update into a pastebin like Github
gist and paste the link here; thanks.

**Comment by smartinsightsfromdata** [comment]
It is here: https://gist.github.com/smartinsightsfromdata/0a299d2df7d4c33337fa

Thanks

**Comment by tdsmith** [comment]
Most of those files should be in your .gitignore. Can you try:

`cd $(brew --repository) && curl -LO https://raw.githubusercontent.com/Homebrew/homebrew/master/.gitignore`?

**Comment by trojjer** [comment]
"Me too."

At least when I installed XCode I finally got `brew install` and `brew doctor` to work again... Sorry, lost the data for that, but the error message was reasonable enough. Thanks.

**Comment by stevegk** [comment]
I got a similar output from brew update and found that Sophos Anti Virus was changing permissions on /usr/local/bin and /usr/local/man/man1 and others every time it updates. A recent commit to brew includes a change to the man page which causes brew update to fail because of the permissions leaving a right old mess. I got out of it by fixing permissions and then `cd $(brew --repository) && git reset --hard origin/master`.
